### PR TITLE
fit-trackee: init at 0.7.15

### PR DESCRIPTION
**Pull request by traxys**
###### Description of changes

[FitTrackee](https://samr1.github.io/FitTrackee/) is a web application to track your outdoor activities (workouts) from gpx files and keep your data on your own server.

To package it I had to package a number of its dependencies.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by traxys** [comment]
Result of `nixpkgs-review pr 227869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/periodiq/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    owner = "bersace";
+    repo = "periodiq";
+    rev = "v${version}";
+    hash = "sha256-Ar0n+Wi1OUtRdhVxrU7Nz4je8ylaHgPZbXE0a30hzU0=";
+  };
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'poetry>=0.12' 'poetry-core' \
+      --replace 'poetry.masonry.api' 'poetry.core.masonry.api'
```
[the remark this answers is not in the document]
Comment (traxys) [reply pkgs/development/python-modules/periodiq/default.nix:27]: I agree, but some of these projects don't seem very active, I'm not sure a release will be fast, I would prefer to first merge this and then do the upstream work

**Comment by traxys** [comment]
Result of `nixpkgs-review pr 227869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/flask-dramatiq/disable_coverage.patch [hunk 2]
```diff
@@ -0,0 +1,8 @@
+diff --git a/tests/unit/pytest.ini b/tests/unit/pytest.ini
+index f44878b..9ad04fd 100644
+--- a/tests/unit/pytest.ini
++++ b/tests/unit/pytest.ini
+@@ -1,2 +1,2 @@
+ [pytest]
+-addopts = -vvv --strict --showlocals --cov=flask_dramatiq --cov-report=term-missing
++addopts = -vvv --strict --showlocals
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flask-dramatiq/disable_coverage.patch:8]: same here

File: pkgs/development/python-modules/watchdog-gevent/disable_coverage.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+diff --git a/setup.cfg b/setup.cfg
+index db37395..3f7fffb 100644
+--- a/setup.cfg
++++ b/setup.cfg
+@@ -1,6 +1,5 @@
+ [tool:pytest]
+ testpaths = tests
+-addopts = --cov watchdog_gevent --cov-report html
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/watchdog-gevent/disable_coverage.patch:8]: and here

**Comment by traxys** [comment]
Result of `nixpkgs-review pr 227869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
For me, `fit-trackee` fails to build (at the end of the tests) with
```
================ 1079 passed, 977 warnings in 270.56s (0:04:30) ================
stopping postgresql
waiting for server to shut down....2023-09-27 13:04:09.038 UTC [1242] LOG:  received fast shutdown request
2023-09-27 13:04:09.043 UTC [1242] LOG:  aborting any active transactions
2023-09-27 13:04:09.044 UTC [1242] LOG:  background worker "logical replication launcher" (PID 1249) exited with exit code 1
2023-09-27 13:04:09.045 UTC [1244] LOG:  shutting down
........................................................... failed
pg_ctl: server does not shut down
```

File: pkgs/development/python-modules/dramatiq/default.nix [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}:
+
+buildPythonPackage rec {
+  pname = "dramatiq";
+  version = "1.14.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "Bogdanp";
+    repo = "dramatiq";
```
Comment (GaetanLepage) [inline pkgs/development/python-modules/dramatiq/default.nix:22]:
```suggestion
{ lib
, stdenv
, buildPythonPackage
, pythonOlder
, fetchFromGitHub
, gevent
, pika
, prometheus-client
, pylibmc
, pytestCheckHook
, redis
, watchdog
, watchdog-gevent
}:

buildPythonPackage rec {
  pname = "dramatiq";
  version = "1.14.2";
  format = "setuptools";
  
  disabled = pythonOlder "3.7";

  src = fetchFromGitHub {
    owner = "Bogdanp";
    repo = "dramatiq";
```

File: pkgs/development/python-modules/flask-dramatiq/default.nix [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+, psycopg2
+, pika
+}:
+
+buildPythonPackage {
+  pname = "flask-dramatiq";
+  version = "0.6.0";
+  format = "pyproject";
+
+  src = fetchFromGitLab {
```
Comment (GaetanLepage) [inline pkgs/development/python-modules/flask-dramatiq/default.nix:22]:
```suggestion
{ lib
, buildPythonPackage
, pythonOlder
, fetchFromGitLab
, poetry-core
, dramatiq
, flask
, requests
, pytestCheckHook
, flask-migrate
, periodiq
, postgresql
, postgresqlTestHook
, psycopg2
, pika
}:

buildPythonPackage {
  pname = "flask-dramatiq";
  version = "0.6.0";
  format = "pyproject";
  
  disabled = pythonOlder "3.6";

  src = fetchFromGitLab {
```

File: pkgs/development/python-modules/periodiq/default.nix [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+, pytest-mock
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "periodiq";
+  version = "0.12.1";
+  format = "pyproject";
+
+  src = fetchFromGitLab {
```
Comment (GaetanLepage) [inline pkgs/development/python-modules/periodiq/default.nix:17]:
```suggestion
{ lib
, buildPythonPackage
, pythonOlder
, fetchFromGitLab
, poetry-core
, dramatiq
, pendulum
, setuptools
, pytest-mock
, pytestCheckHook
}:

buildPythonPackage rec {
  pname = "periodiq";
  version = "0.12.1";
  format = "pyproject";
  
  disabled = pythonOlder "3.5";

  src = fetchFromGitLab {
```

**Review by GaetanLepage — CHANGES_REQUESTED** [review]

**Comment by GaetanLepage** [comment]
Please, rename your commit `python310Packages.staticmap: init at 0.5.5` -> `python310Packages.staticmap: init at 0.5.7`

**Comment by traxys** [comment]
Result of `nixpkgs-review pr 227869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

**Review by GaetanLepage — APPROVED** [review]

File: pkgs/development/python-modules/flask-dramatiq/default.nix [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+  # Does HTTP requests to localhost
+  disabledTests = [
+    "test_fast"
+    "test_other"
+  ];
+
+  meta = with lib; {
+    description = "Adds Dramatiq support to your Flask application";
+    homepage = "https://pypi.org/project/flask-dramatiq/";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flask-dramatiq/default.nix:81]:
```suggestion
    homepage = "https://gitlab.com/bersace/flask-dramatiq";
```

File: pkgs/development/python-modules/flask-dramatiq/default.nix [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  pname = "flask-dramatiq";
+  version = "0.6.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitLab {
+    owner = "bersace";
+    repo = "flask-dramatiq";
+    rev = "840209e9";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flask-dramatiq/default.nix:27]:
```suggestion
    rev = "840209e9bf582b4dda468e8bba515f248f3f8534";
```

File: pkgs/development/python-modules/dramatiq/default.nix [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+    "test_rabbitmq_broker_can_be_passed_a_list_of_uri_for_failover"
+    "test_rabbitmq_broker_can_be_passed_a_semicolon_separated_list_of_uris"
+    "test_rabbitmq_broker_connections_are_lazy"
+    "test_rabbitmq_process_100k_messages_with_cli"
+    "test_rabbitmq_process_10k_fib_with_cli"
+    "test_rabbitmq_process_1k_latency_with_cli"
+  ] ++ (lib.optionals stdenv.isDarwin [
+    # Takes too long for darwin ofborg
+    "test_retry_exceptions_can_specify_a_delay"
+  ]);
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/dramatiq/default.nix:96]:
```suggestion
  ] ++ lib.optionals stdenv.isDarwin [
    # Takes too long for darwin ofborg
    "test_retry_exceptions_can_specify_a_delay"
  ];
```
those are optional

File: pkgs/development/python-modules/dramatiq/default.nix [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+  pythonImportsCheck = [ "dramatiq" ];
+
+  meta = with lib; {
+    description = "Background Processing for Python 3";
+    homepage = "https://pypi.org/project/dramatiq/";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/dramatiq/default.nix:102]:
```suggestion
    homepage = "https://github.com/Bogdanp/dramatiq";
```
The pypi page is usually not that helpful and we don't use it here

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 227869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
Tests seem to fail on x86 darwin. Maybe `export TMP=$(mktemp -d)` will prevent it from trying to access `/tmp`.

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 227869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

**Comment by traxys** [comment]
The darwin runner seems completely overwhelmed, so I'm not sure if the fix is going to help

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 227869` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages failed to build:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
  </ul>
</details>
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

**Comment by natsukium** [comment]
on aarch64-darwin

<details><summary>Error log</summary>

```
flask-dramatiq> pytestCheckPhase
flask-dramatiq> Executing pytestCheckPhase
flask-dramatiq> initializing postgresql
flask-dramatiq> The files belonging to this database system will be owned by user "_nixbld1".
flask-dramatiq> This user must also own the server process.
flask-dramatiq> 
flask-dramatiq> The database cluster will be initialized with locale "en_US.UTF-8".
flask-dramatiq> The default database encoding has accordingly been set to "UTF8".
flask-dramatiq> The default text search configuration will be set to "english".
flask-dramatiq> 
flask-dramatiq> Data page checksums are disabled.
flask-dramatiq> 
flask-dramatiq> creating directory /private/tmp/nix-build-python3.10-flask-dramatiq-0.6.0.drv-0/postgresql ... ok
flask-dramatiq> creating subdirectories ... ok
flask-dramatiq> selecting dynamic shared memory implementation ... posix
flask-dramatiq> selecting default max_connections ... 20
flask-dramatiq> selecting default shared_buffers ... 400kB
flask-dramatiq> selecting default time zone ... UTC
flask-dramatiq> creating configuration files ... ok
flask-dramatiq> running bootstrap script ... 2023-10-18 14:41:32.756 UTC [43190] FATAL:  could not create shared memory segment: Operation not permitted
flask-dramatiq> 2023-10-18 14:41:32.756 UTC [43190] DETAIL:  Failed system call was shmget(key=95894065, size=56, 03600).
flask-dramatiq> child process exited with exit code 1
flask-dramatiq> initdb: removing data directory "/private/tmp/nix-build-python3.10-flask-dramatiq-0.6.0.drv-0/postgresql"
error: builder for '/nix/store/xa7m4pmc55h613q22zq3dq4ryahb6azg-python3.10-flask-dramatiq-0.6.0.drv' failed with exit code 1;
       last 10 log lines:
       > creating subdirectories ... ok
       > selecting dynamic shared memory implementation ... posix
       > selecting default max_connections ... 20
       > selecting default shared_buffers ... 400kB
       > selecting default time zone ... UTC
       > creating configuration files ... ok
       > running bootstrap script ... 2023-10-18 14:41:32.756 UTC [43190] FATAL:  could not create shared memory segment: Operation not permitted
       > 2023-10-18 14:41:32.756 UTC [43190] DETAIL:  Failed system call was shmget(key=95894065, size=56, 03600).
       > child process exited with exit code 1
       > initdb: removing data directory "/private/tmp/nix-build-python3.10-flask-dramatiq-0.6.0.drv-0/postgresql"
       For full logs, run 'nix log /nix/store/xa7m4pmc55h613q22zq3dq4ryahb6azg-python3.10-flask-dramatiq-0.6.0.drv'.
error: 1 dependencies of derivation '/nix/store/m4ras65437y6i6m7qm6q4as3q770ayml-fit-trackee-0.7.22.drv' failed to build
```
</details>

I'm not sure how to fix it.

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 227869` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>fit-trackee</li>
    <li>fit-trackee.dist</li>
    <li>python310Packages.dramatiq</li>
    <li>python310Packages.dramatiq.dist</li>
    <li>python310Packages.flask-dramatiq</li>
    <li>python310Packages.flask-dramatiq.dist</li>
    <li>python310Packages.periodiq</li>
    <li>python310Packages.periodiq.dist</li>
    <li>python310Packages.staticmap</li>
    <li>python310Packages.staticmap.dist</li>
    <li>python310Packages.watchdog-gevent</li>
    <li>python310Packages.watchdog-gevent.dist</li>
    <li>python311Packages.dramatiq</li>
    <li>python311Packages.dramatiq.dist</li>
    <li>python311Packages.flask-dramatiq</li>
    <li>python311Packages.flask-dramatiq.dist</li>
    <li>python311Packages.periodiq</li>
    <li>python311Packages.periodiq.dist</li>
    <li>python311Packages.staticmap</li>
    <li>python311Packages.staticmap.dist</li>
    <li>python311Packages.watchdog-gevent</li>
    <li>python311Packages.watchdog-gevent.dist</li>
  </ul>
</details>

**Comment by wegank** [comment]
> on aarch64-darwin
> 
> Error log
> I'm not sure how to fix it.

With the sandbox enabled?

**Comment by natsukium** [comment]
Yes, `sandbox = true;` in my config.

**Comment by traxys** [comment]
I'm not sure what to do then, should I add the __darwin attribute?

**Review by wegank — APPROVED** [review]

**Comment by natsukium** [comment]
It seems to be a problem with my environment.
The CI results are finally green now, so let's merge.

File: pkgs/servers/geospatial/fit-trackee/default.nix [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+    postgresqlTestHook
+    postgresql
+  ];
+
+  pytestFlagsArray = [
+    "fittrackee"
+  ];
+
+  postgresqlTestSetupPost = ''
+    export DATABASE_TEST_URL=postgresql://$PGUSER/$PGDATABAS?host=$PGHOST
```
Comment (dotlambda) [inline pkgs/servers/geospatial/fit-trackee/default.nix:87]: Is this a typo
```suggestion
export DATABASE_TEST_URL=postgresql://$PGUSER/$PGDATABASE?host=$PGHOST
```
Why do the tests still work?

**Pull request merged**